test/rollout_test.exs

```diff
@@ -0,0 +1,120 @@
+defmodule ConfigCat.RolloutTest do
```

Comment:
testmatrix_input_semantic_2.csv and testmatrix_variationId.csv tests are missing.
https://github.com/configcat/python-sdk/blob/master/configcatclienttests/testmatrix_input_semantic_2.csv
https://github.com/configcat/python-sdk/blob/master/configcatclienttests/testmatrix_variationId.csv

Reply:
Extracted to #19.

Are these new with the v5 config format? They didn't exist in the Ruby SDK when we first wrote our client.